app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -360,97 +311,6 @@ object ConsoleCli {
                 case other => other
               }))
         ),
-      cmd("executedlcremoteunilateralclose")
-        .hidden()
-        .action((_, conf) =>
-          conf.copy(
-            command = ExecuteDLCRemoteUnilateralClose(null,
-                                                      EmptyTransaction,
-                                                      noBroadcast = false)))
-        .text("Executes a unilateral close for the DLC with the given eventId")
-        .children(
-          opt[Sha256DigestBE]("eventid").required
-            .action((eventId, conf) =>
-              conf.copy(command = conf.command match {
-                case executeDLCRemoteUnilateralClose: ExecuteDLCRemoteUnilateralClose =>
-                  executeDLCRemoteUnilateralClose.copy(eventId = eventId)
-                case other => other
-              })),
-          opt[Transaction]("forceCloseTx").required
-            .action((cet, conf) =>
-              conf.copy(command = conf.command match {
-                case executeDLCRemoteUnilateralClose: ExecuteDLCRemoteUnilateralClose =>
-                  executeDLCRemoteUnilateralClose.copy(cet = cet)
-                case other => other
-              })),
-          opt[Unit]("noBroadcast").optional
-            .action((_, conf) =>
-              conf.copy(command = conf.command match {
-                case executeDLCRemoteUnilateralClose: ExecuteDLCRemoteUnilateralClose =>
-                  executeDLCRemoteUnilateralClose.copy(noBroadcast = true)
-                case other => other
-              }))
-        ),
-      cmd("executedlcforceclose")
```

Comment:
is there a new `executedlc` command that should be added that I'm missing?

Reply:
there is `executedlcunilateralclose`, but I will rename to `executedlc`